debezium-testing/debezium-testing-testcontainers/src/main/java/io/debezium/testing/testcontainers/DebeziumContainer.java

```diff
@@ -110,25 +119,49 @@ public String getPauseConnectorURI(String connectorName) {
         return getConnectorURI(connectorName) + "/pause";
     }
 
+    /**
+     * Returns the "/connectors/<connector>/pause" endpoint.
+     */
+    public String getResumeConnectorURI(String connectorName) {
```

Comment:
```suggestion
    public String getResumeConnectorUri(String connectorName) {
```
(It's the style we generally use)

Reply:
Hmmm... all other methods have a similar name. Renaming all?